### PR TITLE
Remove dupplicated words

### PR DESCRIPTION
**Pull request by mikeweiwei**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by neolit123** [comment]
please rebase to fix the kubadm conflict.
/priority backlog

File: plugin/pkg/admission/resourcequota/admission_test.go [hunk 1]
```diff
@@ -744,7 +744,7 @@ func TestAdmitExceedQuotaLimit(t *testing.T) {
 	}
 }
 
-// TestAdmitEnforceQuotaConstraints verifies that if a quota tracks a particular resource that that resource is
```
Comment (apelisse) [inline plugin/pkg/admission/resourcequota/admission_test.go]: I suspect this one's correct

File: staging/src/k8s.io/apimachinery/pkg/api/resource/generated.proto [hunk 2]
```diff
@@ -73,7 +73,7 @@ option go_package = "resource";
 // form, or don't diff.)
 //
 // This format is intended to make it difficult to use these numbers without
-// writing some sort of special handling code in the hopes that that will
```
Comment (apelisse) [inline staging/src/k8s.io/apimachinery/pkg/api/resource/generated.proto]: Also might be correct

File: staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go [hunk 3]
```diff
@@ -76,7 +76,7 @@ import (
 // form, or don't diff.)
 //
 // This format is intended to make it difficult to use these numbers without
-// writing some sort of special handling code in the hopes that that will
+// writing some sort of special handling code in the hopes that will
 // cause implementors to also use a fixed point implementation.
 //
```
Comment (apelisse) [inline staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go]: Also might be correct

**Review by apelisse — COMMENTED** [review]
LGTM after rebase and nits

File: third_party/forked/godep/license.go [hunk 4]
```diff
@@ -42,7 +42,7 @@ func IsLicenseFile(filename string) bool {
 }
 
 // IsLegalFile returns true if the file is likely to contain some type
-// of of legal declaration or licensing information
+// of legal declaration or licensing information
```
Comment (lavalamp) [inline third_party/forked/godep/license.go]: Let's not change the stuff in third_party, it makes the diff noisier.

**Pull request closed without merging**